### DISCUSSION TITLE
Option to force `links:check` to use GET not HEAD

### DISCUSSION THREAD

**Discussion by joepitt91** [Kovah/LinkAce#547] [Ideas & Feature Requests]
I've found that some of my links add correctly but then get incorrectly marked as down by the next check, due to how the remote server responds to HEAD requests vs GET requests.

E.g. libreddit seems to respond with 404 to HEAD requests, but 200 to GET requests for valid pages - so the link adds properly but then goes to down.

It would be good to have an option to force `links:check` to make GET requests instead of HEAD requests (or try a GET request if the HEAD reports a "broken" response before marking the link as broken).

**Comment by Kovah** [comment]
See #366
